.github/workflows/ci.yml

```diff
@@ -4,9 +4,10 @@ jobs:
   test:
     runs-on: ${{ matrix.os }}
     strategy:
+      max-parallel: 4
       matrix:
-        os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9]
+        os: [macos-latest, ubuntu-latest]
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
Why 3.10 is in braces?

Reply:
I'm not sure about that but this way it will be interpreted as a ~float so it will be `3.1`.